src/Spice86.Core/Emulator/Devices/Video/AeonCard.cs

```diff
@@ -0,0 +1,836 @@
+/*
+ * VGA card from Aeon project (https://github.com/gregdivis/Aeon) ported to Spice86.
+ */
+
+using Aeon.Emulator;
+using Aeon.Emulator.Video;
+using Aeon.Emulator.Video.Modes;
+using Aeon.Emulator.Video.Rendering;
+
+using Serilog.Events;
+
+using Spice86.Core.Emulator.Devices.Video.Fonts;
+using Spice86.Core.Emulator.InterruptHandlers;
+using Spice86.Core.Emulator.IOPorts;
+using Spice86.Core.Emulator.Memory;
+using Spice86.Core.Emulator.VM;
+using Spice86.Shared.Interfaces;
+
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Spice86.Core.Emulator.Devices.Video;
+
+public class AeonCard : InterruptHandler, IVideoCard, IVgaCard, IIOPortHandler, IDisposable {
```

Comment:
How about keeping the device part separated from the int handler part? This would allow:
 - To extend DefaultIOPortHandler which has some logic about unhandled ports
 - To boot a bios which provides its own interrupts

Reply:
Done